src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -47,6 +47,12 @@ public CommandResult execute() {
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        ArrayList<String> keywordsLower = new ArrayList<>();
+        for(String aKeyword : keywordsLower) {
+            keywordsLower.add(aKeyword.toLowerCase());
+        }
+        keywords.clear();
+        keywords.addAll(keywordsLower);
```

Comment:
Don't really have to create a new array. Can do this instead: `keywords[i] = keywords[i].toLowerCase()`